test/calculator_test.js

```diff
@@ -0,0 +1,44 @@
+const assert = require('assert');
+const chai = require('chai');
+const calc = require('../calculator')
+
+describe('calculator', function () {
+  describe('#calculate()', function () {
+    it('adds one number', function () {
+      const calculator = calc.create_calculator('+');
+
+      calculator.add_number(16);
+
+      assert.equal(16, calculator.calculate());
+    });
```

Comment:
This test is good and it works, and you've followed the **Arrange Act Assert** layout.

Reply:
Rare to see a purely positive review comment. But I like it!